src/__tests__/incidents/Incidents.test.tsx

```diff
@@ -35,71 +37,58 @@ describe('Incidents', () => {
-      expect(titleUtil.useUpdateTitle).toHaveBeenCalledTimes(1)
-    })
-  })
-
   describe('routing', () => {
     describe('/incidents/new', () => {
-      it('should render the new incident screen when /incidents/new is accessed', async () => {
-        const { wrapper } = await setup([Permissions.ReportIncident], '/incidents/new')
-
-        expect(wrapper.find(ReportIncident)).toHaveLength(1)
+      it('The new incident screen when /incidents/new is accessed', () => {
+        render([Permissions.ReportIncident], '/incidents/new')
+        expect(screen.getByRole('form', { name: /report incident form/i })).toBeInTheDocument()
       })
 
-      it('should not navigate to /incidents/new if the user does not have ReportIncident permissions', async () => {
-        const { wrapper } = await setup([], '/incidents/new')
-
-        expect(wrapper.find(ReportIncident)).toHaveLength(0)
+      it('should not navigate to /incidents/new if the user does not have ReportIncident permissions', () => {
+        const { container } = render([], '/incidents/new')
+        expect(container).toMatchInlineSnapshot(`<div />`)
       })
     })
 
     describe('/incidents/visualize', () => {
-      it('should render the incident visualize screen when /incidents/visualize is accessed', async () => {
-        const { wrapper } = await setup([Permissions.ViewIncidentWidgets], '/incidents/visualize')
-
-        expect(wrapper.find(VisualizeIncidents)).toHaveLength(1)
+      it('The incident visualize screen when /incidents/visualize is accessed', async () => {
+        const { container } = render([Permissions.ViewIncidentWidgets], '/incidents/visualize')
+        await waitForElementToBeRemoved(() => container.querySelector('.css-1jydqjy'))
+        expect(container.querySelector('canvas')).toBeInTheDocument()
       })
 
-      it('should not navigate to /incidents/visualize if the user does not have ViewIncidentWidgets permissions', async () => {
-        const { wrapper } = await setup([], '/incidents/visualize')
-
-        expect(wrapper.find(VisualizeIncidents)).toHaveLength(0)
+      it('should not navigate to /incidents/visualize if the user does not have ViewIncidentWidgets permissions', () => {
+        const { container } = render([], '/incidents/visualize')
+        expect(container).toMatchInlineSnapshot(`<div />`)
       })
     })
 
     describe('/incidents/:id', () => {
-      it('should render the view incident screen when /incidents/:id is accessed', async () => {
-        const { wrapper } = await setup([Permissions.ViewIncident], '/incidents/1234')
-
-        expect(wrapper.find(ViewIncident)).toHaveLength(1)
+      it('The view incident screen when /incidents/:id is accessed', async () => {
+        const { container } = render(
+          [Permissions.ViewIncident],
+          `/incidents/${expectedIncident.id}`,
+        )
+        await waitForElementToBeRemoved(() => container.querySelector('.css-1jydqjy'))
```

Comment:
Same question as https://github.com/HospitalRun/hospitalrun-frontend/pull/2516#discussion_r554047826

Reply:
https://github.com/HospitalRun/hospitalrun-frontend/pull/2516#discussion_r554181998

Fixed (to be merged)